[VRMC_springBone_extended_collider] fallback の章 (#477)
vrm-c/vrm-specification: specification/VRMC_springBone_extended_collider-1.0/README.md

}
```

### Exporter Implemantation

> *This section is non-normative. *

If a collider is defined in the `VRMC_springBone_extended_collider` extension, output a value that will be ignored or approximated to the collider defined in `VRM_springBone`.

#### Fallback: Inside Sphere Collider /  Inside Capsule Collider

> When exporting a sphere/capsule collider that becomes an internal collider, in environments that do not support `VRMC_springBone_extended_collider`, avoid such things as setting the position of the sphere collider far from the origin so that the fallback collider will not have an effect. Please consider measures.

Reviewer: ```suggestion
When exporting a sphere/capsule collider defined as an inside collider, consider having a fallback value such as setting a sphere collider far away from the origin so that the fallback collider does not affect spring bones in environments that do not support `VRMC_springBone_extended_collider`.
```

Below is an example of outputting so that the fallback collider has no effect.

```json

>     // Example of placing a spherical collider with a radius of 0 far away and making it pseudo-ignored in the fallback environment.

Reviewer: ```suggestion
    // Example of placing a sphere collider with a radius of 0 far away and making it pseudo-ignored in the fallback environment.
```

      "colliders": [
        {
          "node": 0,
          "shape": {
            "sphere": {
              "radius": 0.0,
              "offset": [0.0, -10000.0, 0.0]
            }
          },
          "extensions": {
            "VRMC_springBone_extended_collider": {
              "specVersion": "1.0-draft",
              "shape": {
                "sphere": {
                  "radius": 0.5,
                  "offset": [0.0, 0.0, 0.0],
                  "inside": true
                }
              }
            }
          }
        }
      ]
```

#### Fallback: Plane Collider

> Export to be ignored or approximated in the execution environment.

Reviewer: ```suggestion
When exporting a plane collider, consider having a fallback value such as using a sphere collider with a large enough radius so that the fallback collider approximates the behavior of a plane collider in environments that do not support `VRMC_springBone_extended_collider`.

Below is an example of exporting a fallback collider to approximate a plane collider.
```

```json

>     // Example of arranging a spherical collider with a radius of 1000 and approximating a planar collider in a fallback environment.

Reviewer: ```suggestion
    // Example of placing a sphere collider with a radius of 1000 and approximating a plane collider in a fallback environment.
```

}
```

### Exporter Implemantation

> *This section is non-normative. *

> If a collider is defined in the `VRMC_springBone_extended_collider` extension, output a value that will be ignored or approximated to the collider defined in `VRM_springBone`.

Reviewer: ```suggestion
If a collider is defined in the `VRMC_springBone_extended_collider` extension, it is recommended to define a fallback collider in `VRMC_springBone` to ignore or approximate the collider defined in `VRM_springBone_extended_collider`.
```
